lib/networks.ts

```diff
@@ -35,7 +87,9 @@ import {
     explorerUrl: 'https://explorer.xai-chain.net',
     isArbitrum: true,
     isCustom: true,
+    blockTime: 10,
     name: 'Xai',
+    partnerChainIDs: [ChainId.ArbitrumOne],
     partnerChainID: ChainId.ArbitrumOne,
```

Comment:
I'm confused as to the difference between `partnerChainIDs` and `partnerChainID`

Reply:
Again a temporary misnomer. 😞 `partnerChainID` is the parent chain id, and `partnerChainIDs` is the children chains. So in this case, `partnerChainIDs` should be empty.